src/components/TrendingCard/_index.scss

```diff
@@ -35,9 +35,13 @@
     @include focus-outline($type: outline);
   }
 
+  &__title,
+  &__subtitle {
+    @include text-overflow;
+  }
+
```

Comment:
@SimonFinney This keeps with truncation but I think to truly demonstrate it we'd need to add a width to the component's container in the storybook demo as well. And perhaps include some note in the info/documentation that explains that to consumers.

Reply:
That's a good point! It definitely makes sense to add a width to demonstrate the truncation, but I'd rather hold off from including it in the notes as it seems like a higher-level design decision for text in general. Thoughts?

Reply:
If you don't add a width, and you allow users to customize title/subtitle via the knobs, then they can add text that goes off screen pretty easily, depending in their window width. And then they may think something is wrong. 

If you add a width to the container then they will be able to see that value in the story source, as separate from the component. You could go further and put it inside of grid columns, as we've done with other components.

But I think leaving it to just expand off the page like this isn't idea... they'd expect full width components to overflow properly. This one doesn't scroll/overflow properly due to the `text-overflow` mixin, so it needs to demonstrate that.

Reply:
Oh wait, disregard my previous comment -- just saw you added width but weren't sure about adding notes.

That makes sense to me. I've just been leaning more on adding notes lately to help people where something isn't 100% clear with a knob (which seems like the case here, since the width is on the container and without that width, the component doesn't behave normally with typical overflow/scroll... it just keeps going off the page)